src/sample.js

```diff
@@ -0,0 +1,95 @@
+export default {
```

Comment:
I'm just curious, why did you include this information in the sample.js as well as in the index.js?

Reply:
I was trying to import it to `index.js`, but encountered an error. So I ended up abandoning this idea and directly put the object in the file. 

I should delete this file.